.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+branch = True
+source = src/zope/testbrowser
```

Comment:
(Aside: I usually prefer `source = zope.testbrowser`, because I'm under the impression that in that case it doesn't matter if the tests are run directly against the source tree, or against the installed package in a .tox/pyXY/lib/pythonX.Y/site-packages/zope/testbrowser.  I suppose you've tested this configuration and found that it works fine, so feel free to ignore my drive-by comment.)

Reply:
Is there any other reason why to test it against the installed package?
The only reason (and a really good reason indeed) to me seems to be that it's more sementically concise, you see this configuration and know instantly what is tested without knowing src/zope/testbrowser virtually IS zope.testbrowser